### PR TITLE
Add link to Standard maintenance activity

### LINKED ISSUE #701: The activity for Standard maintenance is not linked from activities
The page [Maintenance of the Standard for Public Code](https://about.publiccode.net/activities/standard-maintenance/) is not linked from Activities nor Activities performed in codebase stewardship. I guess it has disappeared during some restructuring.

### PR DESCRIPTION
**Pull request by Ainali**
And adding missing words
Fixes #701

-----
[View rendered activities/index.md](https://github.com/publiccodenet/about/blob/standard-maintenance/activities/index.md)
[View rendered activities/standard-maintenance/index.md](https://github.com/publiccodenet/about/blob/standard-maintenance/activities/standard-maintenance/index.md)

### CODE REVIEW

File: activities/standard-maintenance/index.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ type: index
 
 The Foundation for Public Code and its codebase stewards maintain the [Standard for Public Code](https://standard.publiccode.net).
```
Comment (ElenaFdR) [inline activities/standard-maintenance/index.md:7]: This gives an incomplete view of our Standard maintenance (it's not just community calls.) 
Suggestion: add that we also update the Standard according to CONTRIBUTING and GOVERNANCE. See the Changelog for examples of past Standard maintenance.

Comment (Ainali) [reply activities/standard-maintenance/index.md:7]: That part is not new, and fixing it shouldn't be a blocker for fixing #701. I was already hesitant in adding the lost words in this PR due to scope creep.

Comment (Ainali) [reply activities/standard-maintenance/index.md:7]: I created #776 to address the incompleteness.

**Review by ElenaFdR — APPROVED** [review]
This seems like an important and straightforward improvement.

**Review by bvhme — APPROVED** [review]

**Pull request merged**